frontend/src/routes/(admin)/oauth/authorize/index.tsx

```diff
@@ -52,7 +52,7 @@ export const userLoader = loader$<
 		if (res.status === 302) {
 			throw redirect(302, res.headers.get('location') || '')
 		} else {
-			throw text(res.status, await res.body.text())
```

Comment:
@xtuc I __think__ this is wrong, please let me know 🙂 

I noticed this thanks to my IDE,
currently we get:
![Screenshot 2023-02-08 at 18 18 34](https://user-images.githubusercontent.com/61631103/217617772-7c14e7a7-6ea2-4ac3-8c56-398e0a48bd62.png)

and with an extra `!`
![Screenshot 2023-02-08 at 18 18 49](https://user-images.githubusercontent.com/61631103/217617819-6247047b-7278-492c-a4bb-3cd213ab5833.png)